src/reducers/__tests__/deleteEmptyThought.ts

```diff
@@ -66,23 +67,23 @@ it('do not delete thought with children', () => {
 
 })
 
-it('archive thought with hidden children', () => {
+it(`archive thought with hidden children - arvhive all children in cursor's parent`, () => {
 
   const steps = [
     newThought(''),
-    newSubthought('=one'),
-    cursorBack,
-    deleteEmptyThought,
+    newSubthought('1'),
+    archiveThought({}),
+    deleteEmptyThought
   ]
 
   // run steps through reducer flow and export as plaintext for readable test
   const stateNew = reducerFlow(steps)(initialState())
+
   const exported = exportContext(stateNew, [HOME_TOKEN], 'text/plain')
 
   expect(exported).toBe(`- ${HOME_TOKEN}
   - =archive
-    - ${''/* prevent trim_trailing_whitespace */}
-      - =one`)
+    - 1`)
```

Comment:
We'll want to test multiple children, otherwise it's possible that the test passes for a single child but fails when there are multiple children.